### PR TITLE
Cloudflare capston project

### PR DESCRIPTION
**Pull request by BenMukebo**
> # For this Capstone project , here are what we did to implement that:

> * [x]   **SETTINGS**;
> * Setting up the project by adding all the files for testing;
> * Add a `DefaultSyle.css` file that contains all reusable classes in our project;
> * [x]   **HOME PAGE**;
> * Building Home page in the `index.js` file;
> * Style the home page in the` style.css` file;
> * [x]   **ABOUT PAGE**;
> * Build the About page in  the `about.js`;
> * style the about page in the `AboutStyle.css`;
> * [x]  **ADD JAVASCRIPT**;
> * Implementing the mobile menu in the `index.js` file;
> * implementing my team list dynamically in the `team.js` file;
> * Implementing the animation effects in the `about.js`;
> * Add some animation on our menu in the desktop version and mobile version ;
> * [x]   **DEPLOYMENT**
> * using GitHub pages to deploy my website;
> * Add the valid demo link to our `README` file;
> * Record a loom video giving the overview of our charity website and put the link to this video in the `README` file.
> * Correct all linter errors

> * [Loom Video Link](https://www.loom.com/share/21006ba701894e098ef20806c352decf)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      <article>
+          <h4>Every company has its Cinderella story about those dark times when it has just started hitting the market. Our story takes its beginning in 2013, at a small office with a few like-minded people and bottomless cups of coffee that kept us working throughout the night when we have just started building a company with a strong idea. We strived to build a brand that will contribute to the world with useful products that empower people and make their lives easier.</h4>
+      </article>
+      <div class="info">
+        <p>Please contact us per Email for more informations about Cloudflare Merkle Town Studio Company!</p>
+        <div class="link">
+            <a href="#">cloudflare2017@ccrwanda.org</a>
+        </div>
+      </div>
+    </section>
```
Comment (DeliceLydia) [inline about.html:56]: - [x] Background of this section looks good already but as per requirements you should follow the design from [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) that means colors, font-family, background etc.. but you were free to personalize just contents. on the `about` page background is looking like the design kindly do that even on the `index` file

Comment (DeliceLydia) [reply about.html:56]: original
bs

 your page
page

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+            </li>
+        </ul>
+        <div class="btn-program">
+          <button type="button">Join Fireart comminity </button>
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+        <div class="">
+          
+        </div>
+    </section>
```
Comment (DeliceLydia) [inline index.html:104]: - [ ] For this section the background is looking okay but you can improve it to look like the design check images down:

Design
og

your page
main

Comment (DeliceLydia) [reply index.html:104]: Refer to that but anything else looks dope.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @BenMukebo, 

Amazing job @BenMukebo, I can see the hard work from your project here is my major highlights for your projects:
- Demonstration of your project is good, you explained everything you worked on and you highlighted a piece of codes you implemented that you are proud of
- Used semantic HTML
- Features speakers section is generated dynamically
- PR is descriptive
- Pages are responsive
- README is professional
Good job!

However, there few issues you still need to address but don't worry you almost there keep your head up!

## Required Changes ♻️
_Check the comments under the review._



### Keep Rocking!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (BenMukebo) [reply index.html:104]: Thanks

File: about.html (hunk 1 above)
Comment (BenMukebo) [reply about.html:56]: Hi  DeliceLydia,
Thank you for the review.

Comment (henatan99) [reply about.html:56]: This issue is well addressed. Well done! :fireworks: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @BenMukebo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [x] OPTIONAL: Your PR has a Github build error. Kindly fix it as you can't update your deployed app with this error. :pray: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (BenMukebo) [reply about.html:56]: Thank you very much 👏🏻

**Pull request merged**